org.nickvision.money.gnome/src/application.cpp

```diff
@@ -19,7 +19,13 @@ namespace Nickvision::Money::GNOME
         }
         m_controller->getAppInfo().setChangelog("- Initial Release");
         std::filesystem::path resources{ Aura::getActive().getExecutableDirectory() / (m_controller->getAppInfo().getId() + ".gresource") };
-        g_resources_register(g_resource_load(resources.string().c_str(), nullptr));
+        GError* resourceLoadError{nullptr};
+        GResource* resource = g_resource_load(resources.string().c_str(), &resourceLoadError);
+        if(resourceLoadError != nullptr)
```

Comment:
You can simplify this to `if(resourceLoadError)` which will hit true if `resourceLoadError` is not null.